ratelimits/names_test.go

```diff
@@ -0,0 +1,29 @@
+package ratelimits
+
+import (
+	"testing"
+
+	"github.com/letsencrypt/boulder/test"
+)
+
+func TestIsNameValid(t *testing.T) {
```

Comment:
A test that covers this already exists [here](https://github.com/letsencrypt/boulder/blob/cad7266d86cb703ab97f80cdd0fba558ddf48c0d/ratelimits/limiter_test.go#L56-L63).

Reply:
Thanks, I think I'm going to leave this since it's more specific and stresses the full range.